src/ray/raylet/node_manager.cc

```diff
@@ -2967,6 +2992,11 @@ MemoryUsageRefreshCallback NodeManager::CreateMemoryUsageRefreshCallback() {
           RAY_LOG(INFO) << "Latest 10 worker details:\n"
                         << this->WorkersDebugString(workers, max_to_print);
 
+          rpc::RayErrorInfo task_failure_reason;
+          task_failure_reason.set_error_message(error_log_ss.str());
```

Comment:
Not sure that the raylet-level error message works as as the user-level exception message. I think we should probably tailor the OutOfMemoryError to the task/worker that died.